gateways/http/activities/list_account_activity.go

```diff
@@ -0,0 +1,50 @@
+package activities
+
+import (
+	"errors"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/Vivi3008/apiTestGolang/domain/usecases/activities"
+	"github.com/Vivi3008/apiTestGolang/gateways/http/middlewares"
+	"github.com/Vivi3008/apiTestGolang/gateways/http/response"
+)
+
+var ErrGetTokenId = errors.New("error to get id from token")
+
+type ActivitiesResponse struct {
+	Type      activities.TypeActivity `json:"type"`
+	Amount    int                     `json:"amount"`
+	CreatedAt time.Time               `json:"createdAt"`
```

Comment:
É interessante manter um padrão nas json tags. Vi q vc está misturando camelCase com snake_case às vezes. [Exemplo onde vc usou snake case](https://github.com/Vivi3008/apiTestGolang/blob/master/gateways/http/bills/create_bill.go#L19)
```suggestion
	CreatedAt time.Time               `json:"created_at"`
```